pkg/volume/portworx/portworx.go

```diff
@@ -274,7 +279,10 @@ func (b *portworxVolumeMounter) SetUpAt(dir string, fsGroup *int64) error {
 		return nil
 	}
 
-	if _, err := b.manager.AttachVolume(b); err != nil {
+	attachOptions := make(map[string]string)
+	attachOptions[attachContextKey] = dir
+	attachOptions[attachHostKey] = b.plugin.host.GetHostName()
```

Comment:
Also add the PVC name if available.

Reply:
I dont have access to PVC name in attach function. It is available only in create phase.